detection_rules/rule_loader.py

```diff
@@ -84,21 +123,14 @@ def __init__(self, rules: Optional[List[TOMLRule]] = None):
         self.id_map: Dict[definitions.UUIDString, TOMLRule] = {}
         self.file_map: Dict[Path, TOMLRule] = {}
         self.rules: List[TOMLRule] = []
+        self.deprecated: DeprecatedCollection = DeprecatedCollection()
         self.frozen = False
 
         self._toml_load_cache: Dict[Path, dict] = {}
 
         for rule in (rules or []):
             self.add_rule(rule)
 
-    def __len__(self):
```

Comment:
what's wrong with these methods?

Reply:
the git diff distorts it. They are still there, just inherited from the `BaseCollection`